### PR TITLE
azurerm_data_factory_integration_runtime_azure_ssis - support public_ips, express_custom_setup, package_store, proxy

### PR DESCRIPTION
**Pull request by njuCZ**
supports a bunch of properties for resource "azurerm_data_factory_integration_runtime_azure_ssis"

**notes:**
1. for the `express_custom_setup`, there are four types of polymorphism. I splitted them into four properties.
2. for the `public_ips` property, in the portal, it should be set with size 2.
image

### CODE REVIEW

**Comment by njuCZ** [comment]
image

File: azurerm/internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go [hunk 1]
```diff
@@ -192,6 +203,133 @@ func resourceDataFactoryIntegrationRuntimeAzureSsis() *pluginsdk.Resource {
...
+				},
+			},
+
+			"express_custom_setup": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"env": {
```
Comment (katbyte) [inline azurerm/internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go:221]:
```suggestion
						"environment": {
```

File: azurerm/internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go [hunk 2]
```diff
@@ -416,15 +577,129 @@ func expandDataFactoryIntegrationRuntimeAzureSsisProperties(d *pluginsdk.Resourc
 	return ssisProperties
 }
 
+func expandDataFactoryIntegrationRuntimeAzureSsisProxy(input []interface{}) *datafactory.IntegrationRuntimeDataProxyProperties {
+	if len(input) == 0 || input[0] == nil {
+		return nil
+	}
+	raw := input[0].(map[string]interface{})
+	result := &datafactory.IntegrationRuntimeDataProxyProperties{
```
Comment (katbyte) [inline azurerm/internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go:585]:
```suggestion
	raw := input[0].(map[string]interface{})

	result := &datafactory.IntegrationRuntimeDataProxyProperties{
```

**Review by katbyte — CHANGES_REQUESTED** [review]
Aside from a couple minor comments this looks good

**Comment by njuCZ** [comment]
@katbyte thanks for your suggestions. I have refined this PR

**Review by katbyte — APPROVED** [review]
Thanks @njuCZ - LGTM 🚀

**Pull request merged**